README.md

```diff
@@ -85,96 +169,66 @@ And check with `FeatureService.enabled?("some.nested_feature")`.
 
 Rspec tests can also be tagged with `feature_{name}: true`. This will turn that feature on just for the duration of that test.
 
-### Running the app
-
-You can either run the development task:
-
-```bash
-# Run the foreman dev server. This will also start the frontend dev task
-./bin/dev
-```
+## Configuring GOV.UK Notify
 
-or run the rails server:
+We use [GOV.UK Notify] to send emails from our apps.
 
-```bash
-# Run a local Rails server
-bundle exec rails server
+If you want to test the Notify functionality locally, you will need to get a test API key from the Notify service. Add it as an environment variable under `SETTINGS__GOVUK_NOTIFY__API_KEY` or add it to a local config file:
 
-# When running the server, you can use any of the frontend tasks, e.g.:
-npm run dev
 ```
+# config/settings.local.yml
 
-### Running the tests
-
-```bash
-bundle exec rake
+# Settings for GOV.UK Notify api & email templates
+govuk_notify:
+  api_key: <API key from Notify>
 ```
 
-## Configuration and deployment
+Example emails can be seen locally by visiting `http://localhost:3000/rails/mailers`
 
-The forms-admin app is containerised (see [Dockerfile](https://github.com/alphagov/forms-admin/blob/main/Dockerfile)) and can be deployed however you would normally deploy a containerised app.
+[GOV.UK Notify]: https://www.notifications.service.gov.uk/
 
-If you are planning to deploy to GOV.UK PaaS without using the container, you can see how this runs in our [Deployment CI action](https://github.com/alphagov/forms-admin/blob/main/.github/workflows/deploy.yml).
+## Configuring Sentry
 
-## Logging
+We use [Sentry] to catch exceptions in production apps and alert us.
```

Comment:
"We use [Sentry] to catch and alert us about exceptions in production apps" - does this work better?

Reply:
Yes, much better!